### PR TITLE
Add datadog telemetry

### LINKED ISSUE #1815: datadog telemetry
Hi!

It would be nice if nomad could emit telemetry in datadog statsd format ( https://docs.datadoghq.com/guides/dogstatsd/ ) 

example: `users.online:1|c|@0.5|#country:china`

### PR DESCRIPTION
**Pull request by jippi**
Closes #1815

This is my first go-code ever, so please be kind :)

### CODE REVIEW

File: vendor/github.com/DataDog/datadog-go/statsd/statsd.go [hunk 1]
```diff
@@ -0,0 +1,581 @@
+// Copyright 2013 Ooyala, Inc.
```
Comment (dadgar) [inline vendor/github.com/DataDog/datadog-go/statsd/statsd.go]: Can you inventor the DataDog version and keep Armon's

Comment (jippi) [inline vendor/github.com/DataDog/datadog-go/statsd/statsd.go]: @dadgar i got no idea what any of that mean ;)

**Comment by jippi** [comment]
@dadgar not sure what I'm missing here, but adding

```
telemetry {
  datadog_address = "localhost:8125"
}
```

to my config yields `telemetry -> invalid key: datadog_address`

**Comment by diptanu** [comment]
@jippi You need to add `datadog_address` in Valid keys here - https://github.com/hashicorp/nomad/blob/master/command/agent/config_parse.go#L526

**Pull request merged**